reactDashboard/src/components/DataCharts/DataChartsSelector.jsx

```diff
@@ -1,61 +0,0 @@
-import React from "react";
-import ToggleButton from "react-bootstrap/ToggleButton";
-import ToggleButtonGroup from "react-bootstrap/ToggleButtonGroup";
-import "./DataChartsSelector.css";
-
-import {
-  PERCENTAGE_CASES,
-  DAILY_CASES,
-  DAILY_DEATHS,
-  TOTAL_CASES,
-  TOTAL_DEATHS,
-} from "./DataChartsConsts";
-
-function DataChartsSelector({ chartType, setChartType }) {
-  if (
-    chartType !== PERCENTAGE_CASES &&
-    chartType !== DAILY_CASES &&
-    chartType !== DAILY_DEATHS &&
-    chartType !== TOTAL_CASES &&
-    chartType !== TOTAL_DEATHS
-  ) {
-    throw new Error("Unrecognised chartType: " + chartType);
-  }
-  if (setChartType === null || setChartType === undefined) {
```

Comment:
This validation logic might be of use in the replacement chart selector (along with the associated test cases)

Reply:
Thanks - will look into it

Reply:
see comment above